vector/src/main/java/im/vector/fragments/VectorMessageListFragment.java

```diff
@@ -225,7 +234,8 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
 
     @Override
     public MatrixMessagesFragment createMessagesFragmentInstance(String roomId) {
-        return VectorMessagesFragment.newInstance(roomId);
+        messagesFragment = VectorMessagesFragment.newInstance(roomId);
+        return messagesFragment;
```

Comment:
Why this change?

Reply:
to keep a ref on `messagesFragment` (see https://github.com/dinsic-pim/tchap-android/pull/559/files#diff-fb7f6e89e62c2978d0566a9d5c480ccdR407)